 The program gives error because depending on the values of the parameters a, b, c and d.
 If t1 = 0 is when the program doesn't work, as we cannot make a division when the denominator is 0, so it gives a Zero Division Error. That happens when c and d are the same number. If c and d are different numbers any error happens.
 This happens in the second print (print("Result 2: ", f(0, 2, 3, 3)))
 We can solve the error with a try and except Zero Division Error, so that when this happens the code continuous working.
